modules/nf-core/pretextmap/main.nf

```diff
@@ -0,0 +1,38 @@
+
+process PRETEXTMAP {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::pretextmap=0.1.9 bioconda::samtools=1.16.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-f3591ce8609c7b3b33e5715333200aa5c163aa61:c6242a6c1a522137de7a9e9ff90779ede11cf5c5-0':
+        'quay.io/biocontainers/mulled-v2-f3591ce8609c7b3b33e5715333200aa5c163aa61:c6242a6c1a522137de7a9e9ff90779ede11cf5c5-0' }"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("*.pretext"), emit: pretext
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    samtools \\
+        view \\
+        -h \\
+        $bam | PretextMap \\
+        $args \\
+        -o ${prefix}.pretext
```

Comment:
The commands are slightly different with `sam/bam` vs `pairs` input and both needs to be implemented.
https://github.com/wtsi-hpag/PretextMap

Reply:
Added support for that but somehow it broke my module. Not sure what the problem is as I don't get any informative logs...